h5c/vic/src/vic-webapp/src/app/create-vch-wizard/storage-capacity/storage-capacity.html

```diff
@@ -56,7 +56,7 @@
       <div class="col-xs-3">
 
         <label>
-          Base image size
+          Maximum Container VM image size
```

Comment:
This is wrapping to 2 lines and hampers readability. Checked with Louis and rather than adjusting layout he suggested to change the text to read "Max Container VM image size" to help it some.

Reply:
https://vmware.invisionapp.com/share/87DSS6N2U#/screens

Reply:
Changing Maximum to Max, leaves the word "size" alone, in the second line. Adding a <br>, and a new line-height now, so it looks like the mockups.